src/rawtoaces_util/rawtoaces_util_priv.h

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Contributors to the rawtoaces Project.
+
+#pragma once
+
+#include <filesystem>
+#include <string>
+#include <vector>
+#include <OpenImageIO/imageio.h>
+
+// Contains the declarations of the private functions,
+// exposed here for unit-testing.
+
+namespace rta
+{
+namespace util
+{
+
+bool collect_image_files(
```

Comment:
this is public API, so I guess I can drop it and use one from /include if you want